stubs/reportlab/reportlab/platypus/tables.pyi

```diff
@@ -0,0 +1,120 @@
+from _typeshed import Incomplete
+from abc import abstractmethod
+from collections.abc import Collection, Iterable, Sequence
+from typing import Any, Literal, overload
+from typing_extensions import TypeAlias, Unpack
+
+from reportlab.lib.colors import Color
+from reportlab.lib.styles import PropertySet
+from reportlab.lib.utils import _UNSET_
+from reportlab.platypus.flowables import Flowable, _HAlignment, _VAlignment
+
+__all__ = ("Table", "TableStyle", "CellStyle", "LongTable")
+
+_Color: TypeAlias = Color | list[float] | tuple[float, float, float, float] | tuple[float, float, float] | str | int
+# TODO: consider creating a tagged union of all the possible commands, although
+#       this would restrict us to passing cmds to TableStyle.__init__ in a tuple
+#       since a list would not be able to be inferred correctly
```

Comment:
Could you give an example of one (partial) element of the tagged union in the comment for posteriority? Does it use a TypedDict with a certain key? Or a tuple?